### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by JuanLPalacios**
### Implemented features
- Set up RSpec.
- Create a Request spec each controller.
- for each action checks:
    - is a success
    - renders "index" template
    - includes in response body correct placeholder text

### CODE REVIEW

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @JuanLPalacios, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has errors from Github actions. Please make sure you pass all checks before submitting the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Gemfile [hunk 1]
```diff
@@ -27,6 +27,9 @@ gem 'stimulus-rails'
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'jbuilder'
 
+gem 'ffi'
+gem 'rails-controller-testing'
+gem 'rspec-rails', '~> 6.0.0.rc1'
```
Comment (FrederickMih) [inline Gemfile]: - [optional] Hi @JuanLPalacios, this gem needs to be in the development and test group. This exposes generators and rake tasks without having to type RAILS_ENV=test.

File: spec/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    it 'renders "show" template' do
+      expect(response).to render_template('show')
+    end
+
+    it 'includes in response body correct placeholder text' do
+      expect(response.body).to include('Here is the details of a a given user')
+    end
+  end
+end
```
Comment (FrederickMih) [inline spec/users_controller_spec.rb]: - [optional] Kindly place this file in the directory spec/requests/ as the requirements say. Please do the same for users_controller_spec.rb.

**Review by FrederickMih — APPROVED** [review]
Hi @JuanLPalacios , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**